flycheck.el

```diff
@@ -2357,7 +2399,12 @@ buffer manually.
    (flycheck-mode
     (flycheck-clear)
 
-    (pcase-dolist (`(,hook . ,fn) flycheck-hooks-alist)
+    (pcase-dolist
+        (`(,hook . ,fn)
+         (if flycheck-automatically-display-error-at-point
+             flycheck-hooks-alist
```

Comment:
Shouldn't this conversion be reversed?  If we `flycheck-automatically-...` is `t`, then we want to bind all the hooks, right?

Reply:
yep, you right.